### ISSUE TITLE
osmwebwizard.py and tileget.py - map data not available

### ISSUE THREAD

**Issue by erev-84** [eclipse/sumo#11423]
Hello,

I tried to generate satellite image with the command below but as can be seen from the screenshot below map data not available for the location (35.338534 33.333721)

`python tileGet.py -n osm.net.xml -t 1200`

![image](https://user-images.githubusercontent.com/82845085/186481975-8c0dd1ca-471f-4ef1-aa46-afe7210dbaef.png)

I also checked whether the map data exists on mapper.acme.com, but unfortunately the map data is not available.
https://mapper.acme.com/?ll=35.338534,33.333721&z=18&t=SL&marker0=35.338534,33.333721,

To get the map data, I tried to download the map data from the mapquest. The command I used is as below

`python tileGet.py -n osm.net.xml -t 1200 --url open.mapquestapi.com/staticmap/v4/getmap --key xxxxxxxxxxxxxxxxxxxxxxxxxxxx`

but as can be seen from the screenshots below, the vertical tile lines do not look nice. Is there a way to remove vertical tile lines?

![image](https://user-images.githubusercontent.com/82845085/186487414-8009b3ac-d4df-4cd7-80db-3eb7c59a4c96.png)

![image](https://user-images.githubusercontent.com/82845085/186487333-67202add-1bc4-457b-90d2-9583c75bb1d7.png)

Finally, as a suggestion for enhancement, I think that the download should be canceled when there is no map data available for download with osmwebwizard.py and tileget.py scripts.

**Comment by namdre** [comment]
@behrisch Your commit only addresses the canceling of the download and not the grey lines, right?

**Comment by behrisch** [comment]
Yes, I do not have an idea what to do idea about the grey lines. I think it is a projection artifact and we neither want to reproject the images nor the SUMO network. Of course @erev-84 may want to try different projections to see if this minimizes the effect by giving a different "proj" string when calling netconvert.
